debian/changelog

```diff
@@ -0,0 +1,64 @@
+pitest (1.4.0-1+revpi10+1) buster; urgency=medium
+
+  * Add MIT license in LICENSE.txt
+  * Add piControl dependency, correct Makefile
+  * Change build system to CMake
+  * Add README.md with help output of binary
+  * fix: Add missing man page
+  * Initial debianization
+  * Assign correct license to sources
```

Comment:
Also relevant changes to piTest done in the old Repo are missing:
```
  * piTest: Add Connect 4 as known device
  * Add SPDX headers
  * Replace strncpy by snprintf
  * Fix gcc -Wformat= complaints
```

The are the changes done to the `piTest/` directory sind the last release (see last change to `debian/changelog`).

Reply:
I added the commit mentioned above as changes to the `debian/` directory were previously also included in the changelog.